parquet-tools/src/main/java/org/apache/parquet/tools/read/SimpleReadSupport.java

```diff
@@ -28,8 +28,23 @@
 import org.apache.parquet.schema.MessageType;
 
 public class SimpleReadSupport extends ReadSupport<SimpleRecord> {
+  private final boolean showEmpty;
+
+  public SimpleReadSupport() {
+      super();
```

Comment:
(nit) super() seems unnecessary here and the other constructor, since it automatically happens anyway.